docs/testing/troubleshoot-manifest.md

```diff
@@ -31,14 +31,14 @@ If you didn't use the [Yeoman generator for Office Add-ins](https://www.npmjs.co
 
 1. Install [Node.js](https://nodejs.org/download/).
 
-2. Run the following command in the root directory of your project. Replace `MANIFEST_FILE` with the name of the manifest file.
+2. Run the following command in the root directory of your project. 
 
     ```command&nbsp;line
-    npx office-addin-manifest validate MANIFEST_FILE
+    npm run validate
     ```
 
     > [!NOTE]
-    > If running this command results in the error message "The command syntax is not valid." (because the `validate` command is not recognized), run the following command to validate the manifest (replacing `MANIFEST_FILE` with the name of the manifest file): 
+    > If this command is not available or not working, run the following command instead to force the use of the latest version of the office-addin-manifest tool (replacing `MANIFEST_FILE` with the name of the manifest file):
     >
     > `npx --ignore-existing office-addin-manifest validate MANIFEST_FILE`
```

Comment:
Can we put this command in a code block? I think it'd be nice to have the copy button, but I don't know if that can be parsed inside a note.

```suggestion
    > ```command&nbsp;line
    > npx --ignore-existing office-addin-manifest validate MANIFEST_FILE
    > ```
```

Reply:
That worked. Thanks.